### ISSUE TITLE
RPM Build Error

### ISSUE THREAD

**Issue by rkhmelichek** [aws/amazon-ec2-utils#10]
Hello,

The commit, 9b1ff7d9d745818f82661288afe6489549c44063, has introduced an `rpmbuild` error.

```
rpmbuild -bb --nodeps --define '__python /usr/libexec/platform-python' --define '_topdir /tmp/amazon-ec2-utils/rpmbuild' /tmp/amazon-ec2-utils/rpmbuild/SPECS/amazon-ec2-utils.spec
warning: bogus date in %changelog: Thu Jul 14 2021 Sai Harsha <ssuryad@amazon.com> 1.3.3
error: %changelog not in descending chronological order
```

**Comment by nmeyerhans** [comment]
I think the problem actually originated with a4c04bf61498e9cff0c04460e8d50a2905fe98f5, which erroneously used "1.3.3" in the changelog. That should have been 1.3-3.

**Comment by nmeyerhans** [comment]
Fixed with #11 

And hopefully avoided in the future with #12
